cfme/tests/services/test_myservice.py

```diff
@@ -184,11 +186,11 @@ def test_retire_on_date_for_multiple_service():
     pass
 
 
-@pytest.mark.meta(coverage=[1678123])
-@pytest.mark.manual
+@pytest.mark.meta(automates=[1678123])
 @pytest.mark.ignore_stream('5.10')
 @pytest.mark.tier(2)
-def test_service_state():
+@pytest.mark.parametrize("check", ["provisioned", "un_provisioned"])
```

Comment:
Minor suggestion: please replace `un_provisioned` <==> `unprovisioned`

Reply:
Agree, I'd say it's _optional_ though